### DISCUSSION TITLE
Free tool for automatic translation

### DISCUSSION THREAD

**Discussion by telmob** [givanz/Vvveb#180] [General]
I've started translating the .po file and it was taking a while, so i started searching for a free tol to translate it automatically. AI age... i mean.... So i found that for this funcionality to work with most translators, you need to pay. So i found this free fantastic tool that i'd like to recommend to accelerate the .po file translation. @givanz I think you should also mention it with POEdit.
The tool is https://www.ajexperience.com/po-translator/ and it translates the file automatically. It doesn't identify certain strings with special characters for example, but still, i managed to translate everything accurately in a couple of hours.

Hope it'll help you.

Edit: pt-BR file attached.
[vvveb-pt-br.zip](https://github.com/user-attachments/files/16600979/vvveb-pt-br.zip)

**Comment by givanz** [comment]
Thank you for the translations and tool suggestion.

I tried the tool but on large files it's very slow and it freezes.

I found https://github.com/OzzyCzech/potrans that works without issues using google api.

The .po files was not updated in a while, I will update the .po files with recently added strings.

**Reply by telmob** [reply]
Fantastic! Thanks!

> The .po files was not updated in a while, I will update the .po files with recently added strings.

Yeah, i found just a few missing. Nothing worrysome.
